### PR TITLE
Serialize round 2 output in Participant demo

### LINKED ISSUE #23: Round 2: Serialize round 2 output in Participant CLI demo

### PR DESCRIPTION
**Pull request by natalieesk**
This closes #23

### CODE REVIEW

File: participant/src/round2.rs [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+        logger.log("Identifier:".to_string());
+
+        let mut identifier_input = String::new();
+
+        input.read_line(&mut identifier_input).unwrap();
+
+        let id_value = identifier_input.trim().parse::<u16>().unwrap();
+        let identifier = Identifier::try_from(id_value).unwrap();
+
+        logger.log(format!("Hiding commitment {}:", i));
```
Comment (conradoplg) [inline participant/src/round2.rs:55]: Using `i` here is a bit confusing, my suggestion is to use the `id_value` that was just read

File: participant/src/round2.rs [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+    logger.log("Number of signers:".to_string());
+
+    let mut signers_input = String::new();
+
+    input.read_line(&mut signers_input).unwrap();
+
+    let signers = signers_input.trim().parse::<u16>().unwrap();
+
+    logger.log("You will receive a message from the coordinator, please enter here:".to_string());
```
Comment (conradoplg) [inline participant/src/round2.rs:35]: I was a bit confused by the prompt (I assumed it was a protocol message with the commitments or something like that, not the message to sign). Here's a suggestion, but feel free to change:

```suggestion
    logger.log("Enter the message to sign (received from the coordinator):".to_string());
```

File: participant/src/round2.rs [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+fn encode_signature_response(signature_share: SignatureShare) -> String {
+    let id = hex::encode(signature_share.identifier().serialize());
+    let sig = hex::encode(signature_share.signature().to_bytes());
+    id + &sig
+}
+
+pub fn print_values_round_2(signature: SignatureShare, logger: &mut dyn Logger) {
+    logger.log("Please send the following to the Coordinator".to_string());
+    logger.log(format!(
+        "Signature: {}",
```
Comment (conradoplg) [inline participant/src/round2.rs:102]: Just to be clear it's not the final signature

```suggestion
        "Signature share: {}",
```

**Review by conradoplg — APPROVED** [review]
Looks good, thanks! I left some optional suggestions to make the prompts clearer

File: participant/src/round2.rs (hunk 1 above)
Comment (natalieesk) [reply participant/src/round2.rs]: Done in [679ac65](https://github.com/ZcashFoundation/frost-zcash-demo/pull/44/commits/679ac65694cde77d9786af3790bbfeb1d714c8e5)

**Pull request merged**